src/core/meshcop/joiner.cpp

```diff
@@ -71,6 +71,7 @@ Joiner::Joiner(ThreadNetif &aNetif):
 
 ThreadError Joiner::Start(const char *aPSKd, const char *aProvisioningUrl)
 {
+    otLogFuncEntry();
```

Comment:
I vote for it being the first line of the method, for the reasons I outlined in another comment.